### PR TITLE
MTDSA-4397: Minor wording changes to some allowances field descriptions

### PR DESCRIPTION
**Pull request by jacobbrownlow**

### CODE REVIEW

File: resources/public/api/conf/1.0/schemas/selfAssessment/retrieveTaxCalc/retrieve_allowancesDeductionsReliefs_response.json [hunk 1]
```diff
@@ -7,11 +7,11 @@
     "summary": {
       "title": "summary",
       "type": "object",
-      "description": "Details the high-level allowances, deductions and reliefs for the calculation.",
+      "description": "Details of allowances, deductions and reliefs used in the calculation.",
       "properties": {
         "totalAllowancesAndDeductions": {
           "title": "totalAllowancesAndDeductions",
-          "description": "The total allowances and deductions available to the calculation. This value must be between 0 and 99999999999. No decimals.",
+          "description": "The total allowances and deductions used in the calculation. This value must be between 0 and 99999999999. No decimals.",
```
Comment (david-walters-HMRC) [inline resources/public/api/conf/1.0/schemas/selfAssessment/retrieveTaxCalc/retrieve_allowancesDeductionsReliefs_response.json]: missing  For example: 5000.99

File: resources/public/api/conf/1.0/schemas/selfAssessment/retrieveTaxCalc/retrieve_allowancesDeductionsReliefs_response.json [hunk 2]
```diff
@@ -20,7 +20,7 @@
         },
         "totalReliefs": {
           "title": "totalReliefs",
-          "description": "The total reliefs available to the calculation. This value must be between 0 and 99999999999. No decimals.",
+          "description": "The total reliefs used in the calculation. This value must be between 0 and 99999999999.99. No decimals.",
```
Comment (david-walters-HMRC) [inline resources/public/api/conf/1.0/schemas/selfAssessment/retrieveTaxCalc/retrieve_allowancesDeductionsReliefs_response.json]: same as above

Comment (jacobbrownlow) [inline resources/public/api/conf/1.0/schemas/selfAssessment/retrieveTaxCalc/retrieve_allowancesDeductionsReliefs_response.json]: Need to check this one with BA

File: resources/public/api/conf/1.0/schemas/selfAssessment/retrieveTaxCalc/retrieve_allowancesDeductionsReliefs_response.json (hunk 1 above)
Comment (jacobbrownlow) [reply resources/public/api/conf/1.0/schemas/selfAssessment/retrieveTaxCalc/retrieve_allowancesDeductionsReliefs_response.json]: Example is pulled from "example" attribute below

**Pull request merged**